dali/operators/image/CMakeLists.txt

```diff
@@ -16,9 +16,11 @@
 
 add_subdirectory(color)
 add_subdirectory(crop)
-
 add_subdirectory(convolution)
 add_subdirectory(distortion)
+if (${BUILD_CVCUDA})
```

Comment:
Is this a CMake option? If so, shouldn't it be `if (BUILD_CVCUDA)` ?

Reply:
It should be. Done